src/Xamarin.Android.NamingCustomAttributes/Android.App/ActivityAttribute.cs

```diff
@@ -78,6 +79,8 @@ public ActivityAttribute ()
 
 	public bool MultiProcess { get; set; }
 
+	string IJniNameProviderAttribute.Name => Name ?? "";
```

Comment:
It's probably fine to get us started, but many of these `*Attribute` files are autogenerated (https://github.com/dotnet/android/pull/8781) and these changes will be lost when we update them.

Reply:
can generator.exe generate nullable compatible code? Or is that quite a huge job to support that? (I would assume the latter).

Reply:
It looks like these already have `*.Partial.cs`, so I'll move them there.